### DISCUSSION TITLE
How to maintain scroll position on navigate

### DISCUSSION THREAD

**Discussion by maminskik** [livewire/livewire#9024] [1. Help] [answered]
Im using livewire and want to prevent the scroll position from going to the top when navigating using wire:navigate.
Livewire seems to reset the scroll position, which is not what I want. I would like to keep the scroll position the same for aside element after the navigation completes.

**Comment by danie-ramdhani** [comment]
You need to persist your `aside` element, then add `wire:scroll` attribute. https://livewire.laravel.com/docs/navigate#preserving-scroll-position

**Comment by joshhanley** [comment, marked as answer]
@maminskik We're in the process of investigating all wire:navigate issues and I believe this has been fixed in PR #9341 which will be available in the next release, so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
